components/VcItemContent.tsx

```diff
@@ -138,38 +149,36 @@ export const VcItemContent: React.FC<VcItemContentProps> = props => {
                 <Text
                   testID="fullNameValue"
                   weight="semibold"
-                  size="smaller"
                   style={
                     !props.verifiableCredential
                       ? Theme.Styles.loadingTitle
-                      : Theme.Styles.subtitle
+                      : Theme.Styles.detailsValue
                   }>
                   {fullName}
                 </Text>
               </Column>
 
-              <Column margin="10 0 0 0">
+              <Column>
                 <Text
                   testID="idType"
                   color={
                     !props.verifiableCredential
                       ? Theme.Colors.LoadingDetailsLabel
                       : Theme.Colors.DetailsLabel
                   }
-                  weight="semibold"
-                  size="smaller"
+                  weight="regular"
+                  style={Theme.Styles.titleTextSize}
```

Comment:
Even here with `Theme.TestStyles.Header`

Reply:
Going with smaller one since the font size matches the wireframes